.renovaterc.json

```diff
@@ -106,10 +106,10 @@
         "^@schematics",
         "angular",
         "^@ngrx",
-        "^zone.js"
-      ],
-      "excludePackagePatterns": [
-        "^@nrwl"
+        "^zone.js",
+        "^@nrwl",
+        "^@nX",
```

Comment:
is it case sensitive ? the scope seems to be `@nx` in our package.json, same for `nx`

Reply:
oops, it is not case sensitive but it's a typo :'(